side-products/merkl.md

```diff
@@ -0,0 +1,72 @@
+---
+description: Merkl is a improved incentive mechanism for UniswapV3-like positions built by Angle Labs
+---
+
+# 🌲 Merkl, by Angle
+
+Merkl is a mechanism to incentivize UniswapV3-type liquidity positions in a flexible and efficient way. It is built and maintained by Angle Labs, but is separate from Angle Protocol.
```

Comment:
to incentivize liquidity on AMMs that use concentrated liquidity (like UniswapV3)